tests/node-17.0/tests.js

```diff
@@ -16,7 +16,7 @@ module.exports = async (req, res) => {
     const payload = JSON.parse(req.payload === '' ? '{}' : req.payload);
 
     const todo = await fetch(`https://jsonplaceholder.typicode.com/todos/${payload.id ?? 1}`).then(r => r.json());
-
+    console.log('stdout', 'other outputs');
```

Comment:
In test, can we add multiple logs? also multiple types, maybe even try to log object. For all, lets have test check in PHP

Reply:
haven't gone to the test yet, still trying to get the runtimes right before moving on to test